### ISSUE TITLE
Mobile menu toggle button fails Lighthouse accessibility test:  Buttons do not have an accessible name

### ISSUE THREAD

**Issue by joethseo** [soflyy/breakdance-bugs#374]
See here:

<img width="947" alt="Screenshot of Google Chrome (13-09-2022, 09-50-28)" src="https://user-images.githubusercontent.com/66306824/189857483-e50b14b2-9f2f-48c1-87f2-0d0336b2a48b.png">

This was taken from this URL:

https://web.dev/measure/?gclid=CjwKCAjwsfuYBhAZEiwA5a6CDPPKsunL1SImhOMv7nXi_zV1M8Nl-gWo_721IjMaCuC-ZwB2n_EfrhoC2OEQAvD_BwE&url=https%3A%2F%2Fwww.tribeseo.com%2F#

**Comment by louisreingold** [comment]
Thanks for this report!

We did all sorts of A11Y enhancements to the menu in Breakdance 1.3.

If you find any further A11Y problems, please report them.
